src/modules/organisation/api/organisation-query.param.ts

```diff
@@ -0,0 +1,35 @@
+import { AutoMap } from '@automapper/classes';
```

Comment:
Rename file to find-organisation-query.params.ts

Reply:
✅